projects/sdk/src/lib/silo/Withdraw.ts

```diff
@@ -42,10 +43,10 @@ export class Withdraw {
 
     if (seasons.length === 1) {
       Withdraw.sdk.debug("silo.withdraw(): withdrawDeposit()", { address: token.address, season: seasons[0], amount: amounts[0] });
-      contractCall = Withdraw.sdk.contracts.beanstalk.withdrawDeposit(token.address, seasons[0], amounts[0]);
+      contractCall = Withdraw.sdk.contracts.beanstalk.withdrawDeposit(token.address, seasons[0], amounts[0], toMode);
     } else {
       Withdraw.sdk.debug("silo.withdraw(): withdrawDeposits()", { address: token.address, seasons: seasons, amounts: amounts });
-      contractCall = Withdraw.sdk.contracts.beanstalk.withdrawDeposits(token.address, seasons, amounts);
+      contractCall = Withdraw.sdk.contracts.beanstalk.withdrawDeposits(token.address, seasons, amounts, toMode);
```

Comment:
Shouldn't these pass stem instead of season?

Reply:
I'll follow up with a separate PR to do an exhaustive season -> stem renaming where applicable, figured it'd be too much for the scope here; this simply gets contract funcs back up to speed